tests/conftest.py

```diff
@@ -31,6 +33,19 @@ def alert_dict():
     }
 
 
+@pytest.fixture(scope='session', autouse=True)
```

Comment:
We should avoid `autouse` as it causes [confusion](https://github.com/alphagov/notifications-template-preview/commit/b9a3ace8c675ba3d50869782f14c4ce73a495b3a). Can we just list it in the small number of tests that require it?

Reply:
Removed in 5a37bbb